src/auto/builder_scope.rs

```diff
@@ -0,0 +1,28 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use glib::object::IsA;
+use glib::translate::*;
+use gtk_sys;
+use std::fmt;
+
+glib_wrapper! {
+    pub struct BuilderScope(Interface<gtk_sys::GtkBuilderScope>);
```

Comment:
This will need some subclass defined here in gtk so that builder can automatically connect signals/etc. Best to create an issue for that.

Reply:
Done https://github.com/gtk-rs/gtk4/issues/24